node/src/types/json_compatibility/execution_result.rs

```diff
@@ -18,11 +19,37 @@ use casper_execution_engine::{
     },
     shared::{stored_value::StoredValue, transform::Transform as EngineTransform},
 };
-use casper_types::{CLValue, U128, U256, U512};
+use casper_types::{
+    bytesrepr::{self, FromBytes, ToBytes},
+    CLValue, U128, U256, U512,
+};
 
 #[cfg(test)]
 use crate::testing::TestRng;
 
+/// Constants to track operation serialization.
+pub const OP_READ_TAG: u8 = 1;
+pub const OP_WRITE_TAG: u8 = 2;
+pub const OP_ADD_TAG: u8 = 3;
+pub const OP_NOOP_TAG: u8 = 4;
```

Comment:
None of these should be `pub`.  Also, let's zero-index them as per other cases.

Reply:
This wasn't actioned, but you marked the issue resolved!

Reply:
WELP